### ISSUE TITLE
Minimizing restored group creates new group

### ISSUE THREAD

**Issue by generic-humanoid** [suhay/vscode-editor-group-minimizer#20]
Perhaps I'm doing something wrong but, if I've minimized an editor group, restore it, then subsequently minimize it again, this creates a new group. This is problematic if the originally minimized group has been given a name. Desired behavior would be to minimize this group back into its previously named group without creating a new group.

**Comment by braselog** [comment]
Agreed. It would be nice to have a "re/minimize" button in addition to the "remove" and "restore" buttons under the "MINIMIZED GROUPS" section of the Explorer. This would let you easily switch between groups of tabs.
